botocore/tokens.py

```diff
@@ -183,7 +183,9 @@ class SSOTokenProvider:
     _GRANT_TYPE = "refresh_token"
     DEFAULT_CACHE_CLS = JSONFileCache
 
-    def __init__(self, session, cache=None, time_fetcher=_utc_now):
+    def __init__(
+        self, session, cache=None, profile_name=None, time_fetcher=_utc_now
```

Comment:
Whenever adding new arguments to a signature, we should add it to the end. Otherwise, it would break backwards compatibility if it was called using purely positional arguments. While this class is not considered a public interface, it is still good practice to not break something unless there is a strong reason to.